libs/blocks/preflight/preflight.css

```diff
@@ -861,7 +861,7 @@ img:hover ~ .asset-meta,
   display: grid;
   grid-template-columns: 1fr 1fr;
   gap: 48px;
-  height: 250px;
+  height: 280px;
```

Comment:
I'm still not sure why we need a fixed value here instead of letting things flow naturally, but I guess this does address the issue for now.